osd_architecture/osd_policy/osd-sre-access.adoc

```diff
@@ -4,4 +4,8 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 :context: osd-sre-access
 [id="osd-sre-access"]
 = SRE and service account access
-////
\ No newline at end of file
+
```

Comment:
I think the first set of //// is causing the build failure

Reply:
Hi @bmcelvee,  I think you are correct :( Just got off my 1:1 with Andrew, working on this now....